components/cqrs/cqrs_test.go

```diff
@@ -224,3 +226,126 @@ func (p *publisherStub) Publish(topic string, messages ...*message.Message) erro
 
 	return nil
 }
+
+func TestFacadeConfig_Validate(t *testing.T) {
+	ts := NewTestServices()
+
+	router, err := message.NewRouter(message.RouterConfig{}, ts.Logger)
+	require.NoError(t, err)
+
+	validConfig := cqrs.FacadeConfig{
+		GenerateCommandsTopic: func(commandName string) string {
+			return commandName
+		},
+		GenerateEventsTopic: func(eventName string) string {
+			return eventName
+		},
+		CommandHandlers: func(cb *cqrs.CommandBus, eb *cqrs.EventBus) []cqrs.CommandHandler {
+			return []cqrs.CommandHandler{}
+		},
+		EventHandlers: func(cb *cqrs.CommandBus, eb *cqrs.EventBus) []cqrs.EventHandler {
+			return []cqrs.EventHandler{}
+		},
+		Router:            router,
+		CommandsPublisher: ts.CommandsPubSub,
+		CommandsSubscriberConstructor: func(handlerName string) (message.Subscriber, error) {
+			return ts.CommandsPubSub, nil
+		},
+		EventsPublisher: ts.EventsPubSub,
+		EventsSubscriberConstructor: func(handlerName string) (message.Subscriber, error) {
+			return ts.EventsPubSub, nil
+		},
+		Logger:                ts.Logger,
+		CommandEventMarshaler: ts.Marshaler,
+	}
+
+	testCases := []struct {
+		Name   string
+		Config cqrs.FacadeConfig
+		Valid  bool
+	}{
+		{
+			Name:   "valid",
+			Config: validConfig,
+			Valid:  true,
+		},
+		{
+			Name: "missing_GenerateCommandsTopic",
+			Config: transformConfig(validConfig, func(config *cqrs.FacadeConfig) {
+				config.GenerateCommandsTopic = nil
+			}),
+			Valid: false,
+		},
+		{
+			Name: "missing_GenerateCommandsTopic",
```

Comment:
you forgot to change the test case names :poultry_leg:

Reply:
:man_facepalming: f9f87f3